src/net/toxdns.cpp

```diff
@@ -251,7 +251,7 @@ QString ToxDNS::queryTox3(const tox3_server& server, const QString &record, bool
     QMessageBox::StandardButton btn = QMessageBox::warning(nullptr, "qTox", tr("It appears that qTox has to use the old tox1 protocol to access DNS record of your friend's Tox ID.\n\
 Unfortunately tox1 is not secure, and you are at risk of someone hijacking what is sent between you and ToxDNS service.\n\
 Should tox1 be used anyway?\n\
-If unsure, press “No”, so that request to ToxDNS service will not be made using unsecure protocol."), QMessageBox::Yes|QMessageBox::No, QMessageBox::No);
+If unsure, press 'No', so that request to ToxDNS service will not be made using unsecure protocol."), QMessageBox::Yes|QMessageBox::No, QMessageBox::No);
     if (btn == QMessageBox::Yes)
         queryTox1(record, silent);
 
```

Comment:
Just a little tip: A string can be split in c++ to several lines like this …

```c++
    QMessageBox::StandardButton btn = 
        QMessageBox::warning(nullptr, "qTox", 
                             tr("It appears that qTox has to use the old protocol to access DNS record of your friend's Tox ID.\n"
                                "Unfortunately tox1 is not secure, and you are at risk of someone hijacking what is sent between you and ToxDNS service.\n"
                                "Should tox1 be used anyway?\n"
                                "If unsure, press 'No', so that request to ToxDNS service will not be made using unsecure protocol."),
                                QMessageBox::Yes|QMessageBox::No, QMessageBox::No);
```

This way the editor makes better use of indentations. Same applies to lines from 305.